README.md

```diff
@@ -48,6 +48,14 @@ No faster way to get started than by diving in and playing around with one of ou
 
 Need specific sample code to help get started with Redis? Start here.
 
+## Getting started with Redis & Vector Search
+
+| Recipe | Description |
+| --- | --- |
+| [/redis_intro](python-recipes/redis-intro/redis_intro.ipynb) | The place to start if brand new to Redis |
```

Comment:
I went back and forth on that. I opted without number for this case because you could start from either they are not ordered but I agree does look better... I'll change to numbers.

And yeah I can a few full text search examples as well!